### PR TITLE
Issue 1189 use test tld

### LINKED ISSUE #1189: Use .test TLD
According to @seisvelas 

Very small nitpick: when using test accounts that aren't real, like expired@denbond7.com, it's better to use a .test TLD. If there were ever an unexpected side-effect of the code whereby the owner of expired@denbond7.com could influence what happens somehow, someone could buy the domain denbond7.com and make that happen.

Unlikely, I know, but not unimaginable, and the solution is to simply use .test TLD, so why not?

.test TLD is reserved by IETF and will never be installed on the global DNS, so it's safer.

Eventually I'm going to review this codebase for exactly this and switch everything to .test TLD test accounts anyway. But just note going forward that .test TLD test accounts are preferable :)

### PR DESCRIPTION
**Pull request by DenBond7**
This PR replaced 'denbond7.com' with 'flowcrypt.test'
close #1189 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities

### CODE REVIEW

**Comment by DenBond7** [comment]
@seisvelas Please check maybe I've missed something

**Review by IvanPizhenko — CHANGES_REQUESTED** [review]
I think some changes can be done.

File: FlowCrypt/src/androidTest/resources/ssl/localhost-cert.pem [hunk 1]
```diff
@@ -0,0 +1,87 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number:
+            09:b7:10:b7:71:2c:10:36:01:54:f9:6d:1b:21:d4:7e:36:f1:e9:54
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: C=UA, ST=Debug, O=FlowCrypt, CN=flowcrypt.test/emailAddress=admin@flowcrypt.test
```
[the remark this answers is not in the document]
Comment (tomholub) [reply FlowCrypt/src/androidTest/resources/ssl/localhost-cert.pem:7]: Agree.. :)

[the remark this answers is not in the document]
Comment (DenBond7) [reply FlowCrypt/src/androidTest/resources/ssl/localhost-cert.pem:7]: Got it. Will do

**Review by IvanPizhenko — APPROVED** [review]

**Pull request merged**